CMake/fletch-tarballs.cmake

```diff
@@ -290,7 +290,8 @@ list(APPEND fletch_external_sources libkml)
 
 # Qt
 # Support 4.8.6 and 5.10 optionally
-if (fletch_ENABLE_Qt OR fletch_ENABLE_ALL_PACKAGES)
+if (fletch_ENABLE_Qt OR fletch_ENABLE_VTK OR fletch_ENABLE_qtExtensions OR
```

Comment:
I don't understand allowing a Qt version being set if Qt is not being built.

Reply:
This is so users can use their system Qt, but still select which Qt version dependent packages build against.